### ISSUE TITLE
Fails to load on servers

### ISSUE THREAD

**Issue by FireEgl** [iamshift/BigExtras#7]
It fails to load on servers..
```
[13:52:16] [main/FATAL] [minecraft/Main]: Failed to start the minecraft server
net.minecraftforge.fml.LoadingFailedException: Loading errors encountered: [
        Big Extras (bigextras) has failed to load correctly
§7java.lang.RuntimeException: Attempted to load class com/iamshift/bigextras/BigExtrasClient for invalid dist DEDICATED_SERVER
]
        at net.minecraftforge.fml.ModLoader.waitForTransition(ModLoader.java:263) ~[forge:?] {re:classloading}
        at net.minecraftforge.fml.ModLoader.dispatchAndHandleError(ModLoader.java:230) ~[forge:?] {re:classloading}
        at net.minecraftforge.fml.ModLoader.gatherAndInitializeMods(ModLoader.java:188) ~[forge:?] {re:classloading}
        at net.minecraftforge.fml.server.ServerModLoader.load(ServerModLoader.java:44) ~[forge:?] {re:classloading}
        at net.minecraft.server.Main.main(Main.java:95) ~[?:?] {re:classloading,re:mixin,pl:mixin:A}
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_282] {}
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_282] {}
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_282] {}
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_282] {}
        at net.minecraftforge.fml.loading.FMLServerLaunchProvider.lambda$launchService$0(FMLServerLaunchProvider.java:51) ~[forge-1.16.5-36.1.31.jar:36.1] {}
        at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-8.0.9.jar:?] {}
        at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54) [modlauncher-8.0.9.jar:?] {}
        at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72) [modlauncher-8.0.9.jar:?] {}
        at cpw.mods.modlauncher.Launcher.run(Launcher.java:82) [modlauncher-8.0.9.jar:?] {}
        at cpw.mods.modlauncher.Launcher.main(Launcher.java:66) [modlauncher-8.0.9.jar:?] {}
        at net.minecraftforge.server.ServerMain$Runner.runLauncher(ServerMain.java:63) [forge-1.16.5-36.1.31.jar:?] {}
        at net.minecraftforge.server.ServerMain$Runner.access$100(ServerMain.java:60) [forge-1.16.5-36.1.31.jar:?] {}
        at net.minecraftforge.server.ServerMain.main(ServerMain.java:57) [forge-1.16.5-36.1.31.jar:?] {}
```

**Comment by shiftthedev** [comment]
fixed in [1.5.3](https://www.curseforge.com/minecraft/mc-mods/big-extras-forge/files/3384514)

**Comment by Mattabase** [comment]
Any chance this fix could be backported to the 1.17 Fabric versions? I am waiting on a few mods to update my server to 1.17.1 and would love to not have to wait to use both small and big extras on my server

I admit I did not test if it crashes on servers in 1.17 before posting this. Appears it does not crash, nevermind

**Comment by shiftthedev** [comment]
I also forgot to mark them as 1.17 compatible, but it's fixed now.
